src/blocks/common.blocks/block-example/block-example.bemtree

```diff
@@ -75,6 +75,8 @@ block('block-example')(
                     },
                     {
                         block: 'dropdown',
+                        js: { url: 'https://disk.yandex.net/qr/?format=svg&text=' +
```

Comment:
I think it would be much better to have this param in `block-example__qr`. You can extract them within the block with `this.elemParams(this.elem('qr')).url`.